src/pages/listpage/CardListPage.jsx

```diff
@@ -0,0 +1,78 @@
+import { RecipientsAPI } from 'data/CallAPI';
+import { useState, useEffect } from 'react';
+import CardList from 'components/card/CardList';
+import TextDropdownField from 'components/textfield/TextDropdownField';
+import TextInputField from 'components/textfield/TextInputField';
+import EmojiToggle from 'components/EmojiToggle';
+import "pages/listpage/CardListPage.scss";
+
+function HotSort(recipient) {
+  if (!recipient || !Array.isArray(recipient.results)) {
+    //레시피언트 없는지와 레시피언트리절트가 배열인지 확인
+    return [];
+  }
+  return [...recipient.results].sort((a,b) => {
+    if(b.messageCount === a.messageCount) {
+      return b.reactionCount - a.reactionCount;
+    }
+    return b.messageCount - a.messageCount;
+  });
+}
+
+function DateSort(recipient) {
+  if (!recipient || !Array.isArray(recipient.results)) {
+    //레시피언트 없는지와 레시피언트리절트가 배열인지 확인
+    return [];
+  }
+  return [...recipient.results].sort((a,b) => b.createdAt - a.createdAt);
+}
+
+function CardListPage() {
+  const test = ['a', 'b', 'c', 'd'];
+  const [recipient, setRecipient] = useState({});
```

Comment:
```suggestion
const [recipients, setRecipients] = useState({});
```
recipient는 하나가 아니라 여러개 받아오니 s가 붙는게 좋을거 같아용